src/generators/app.ts

```diff
@@ -491,25 +498,28 @@ class App extends Generator {
     }
     if (this.ts) {
       dependencies.push(
-        'tslib@1',
+        'tslib@^1',
       )
       devDependencies.push(
-        '@types/chai@^4',
-        '@types/mocha@^5',
         '@types/node@^10',
-        'typescript@^3.0',
-        'ts-node@^7',
+        'typescript@^3.3',
```

Comment:
I'm wary about careting ts -- they sometimes introduce breaking changes in minor releases.

Reply:
Sure